README.md

```diff
@@ -8,6 +8,11 @@
 
 Easy i18n localization for Laravel, an useful tool to combine with Laravel localization classes.
 
+This documentation covers version 3.x, compatible exclusively with Laravel 9 and later.
+We strongly recommend using this version, as it resolves numerous compatibility issues present in versions 1.x and 2.x, 
+and seamlessly integrates with Laravel's default caching. 
+If you need to use an older version, please switch to `legacy` branch by clicking [here](https://github.com/mcamara/laravel-localization/tree/legacy).
+
```

Comment:
Upgrade 3.x introduces significant changes, and my goal is to maintain a clean and concise README for this version. Including setup details for 1.x-2.x alongside 3.x would unnecessarily bloat the README, as the setups differ substantially

My idea was to streamline the README by focusing exclusively on 3.x, while users requiring older versions could switch to the 2.x or legacy branch. These branches would retain the current README from master, similar to how Laravel organizes branches for each major version (e.g., 11.x, 10.x).

Since we're also removing files like caching-related documents in 3.x, I thought this approach would make it more convenient for users sticking with an outdated Laravel and version 2.x.

However, if this doesn’t resonate, I’m open to an alternative: creating a separate `LEGACY_README.md` for installation instructions and other details for 1.x-2.x. My primary objective is to keep the 3.x README clean and focused.

Reply:
@iwasherefirst2 I don't like the idea of having a legacy readme file. It goes against the principles of git , as it's just a copy of an older version. Instead, I've gone ahead and created branches for all major versions, 0.x, 1.x, 2.x, etc and referred to those from the readme itself. This way, we refer to the entire repository and documentation for the specific version.

Reply:
I agree, that is the best option. Thanks!